Include/internal/pycore_tracemalloc.h

```diff
@@ -94,6 +94,12 @@ struct _tracemalloc_runtime_state {
     /* domain (unsigned int) => traces (_Py_hashtable_t).
        Protected by TABLES_LOCK(). */
     _Py_hashtable_t *domains;
+    /* Number of references created.
```

Comment:
The terminology here is a bit confusing, but I don't think we should be calling these "references" or "refs". It's tracking object allocation and deallocation, not references or reference counts.

(You can create lots of references to the same object and tracemalloc doesn't track that.)